pkg/util/util.go

```diff
@@ -251,20 +252,12 @@ func GetPdClient(etcdURLs string, securityConfig security.Config) (pd.Client, er
 		return nil, errors.Trace(err)
 	}
 
-	pdReconnTimes := 30
-
 	var pdCli pd.Client
-	for i := 1; i < pdReconnTimes; i++ {
```

Comment:
Removing retries may reduce usability, as long as drainer can't connect to pd, it will try the relay log and then exit.

Reply:
we can leave the simple retry logic

Reply:
As the commen added, the pd.NewClient will retry 100 time already.

Reply:
now it's reasonable